e2e/skills/test_skills_crud.py

```
@@ -26,103 +27,432 @@ def test_skills_crud():
+            print("   [OK] Edit modal opened")
+
+            # Update to test name
+            skill_input.fill(test_skill_name)
+            page.wait_for_timeout(200)
+
+            # Update order
+            order_input = page.locator('input[placeholder="Order"]').last
+            order_input.fill("88")
+            page.wait_for_timeout(200)
+
+            page.screenshot(path="/tmp/skills_12_skill_edit_filled.png")
```

Ruff (0.14.4)

Probable insecure usage of temporary file or directory: "/tmp/skills_12_skill_edit_filled.png"

(S108)